### ISSUE TITLE
[Freeze] Copy-pasting frozen cells does not preserve color

### ISSUE THREAD

**Issue by SteliosStefanatos** [ipython-contrib/jupyter_contrib_nbextensions#1146]
As the title says, when I do the standard Jupyter procedure to copy of a frozen cell either to the same notebook or a different one, the new cell retains its "frozen attributes" but without the color (i.e., it appears as a normal cell).

**Comment by jcb91** [comment]
yeah, this is a result of the `create.Cell` event firing *before* the cellis reconstructed using its `fromJSON` method -  the same thing happens with "undeleting" cells. Personally, I think there should be another `fromJSON.Cell` event we could catch, but so far there isn't. In the meantime I guess we could patch the `fromJSON` method to trigger an event, but this will need doing for a whole bunch of nbextensions, and I'm not immediately sure how best to avoid patching the method multiple times...
